Dockerfile

```diff
@@ -1,29 +1,32 @@
-# Use official Node.js image as the base image
-FROM node:18
+#
+# Builder
+#
+FROM node:18-alpine AS builder
 
-# Set the working directory inside the container
-WORKDIR /usr/src/app
+WORKDIR /app
 
-# Install pnpm
-RUN npm install -g pnpm
-
-# Copy the rest of the application code to the working directory
 COPY . .
 
-# Install workspace dependencies
-RUN pnpm install
+RUN apk add --update --upgrade --no-cache \
+        alpine-sdk \
+        python3 \
+        ca-certificates \
+    && \
+    npm install -g pnpm@v8.6.10 && \
```

Comment:
just a Q: why pin the version? wouldn't we forget to update pnpm version here?

Reply:
Good call. If you use a different one, you will see a warning. We specify the `packageManager` version in the `package.json` file.

I'm happy to bump it and use the latest NodeJS LTS. What do you think?

Reply:
sounds good, please proceed